integrated_channels/moodle/models.py

```diff
@@ -106,8 +109,92 @@ def channel_code():
         """
         return 'MOODLE'
 
+    def get_learner_data_exporter(self, user):
+        return MoodleLearnerExporter(user, self)
+
+    def get_learner_data_transmitter(self):
+        return MoodleLearnerTransmitter(self)
+
     def get_content_metadata_exporter(self, user):
         return MoodleContentMetadataExporter(user, self)
 
     def get_content_metadata_transmitter(self):
         return MoodleContentMetadataTransmitter(self)
+
+
+@python_2_unicode_compatible
+class MoodleLearnerDataTransmissionAudit(models.Model):
+    """
+    The payload we send to Moodle at a given point in time for an enterprise course enrollment.
+
+    """
+    moodle_user_email = models.CharField(max_length=255, blank=False, null=False)
+
+    enterprise_course_enrollment_id = models.PositiveIntegerField(blank=False, null=False, db_index=True)
+    course_id = models.CharField(max_length=255, blank=False, null=False)
+    course_completed = models.BooleanField(default=True)
+    grade = models.CharField(max_length=100, blank=False, null=False)
```

Comment:
hmm, grade max_length 100 chars? too long?

Reply:
🤦 haha yeah, that's probably supposed to be an int/decimal, not even a char